sunpy/data/fits_solarnet_schema.yaml

```diff
@@ -0,0 +1,307 @@
+  required: false
+  data_type: str
+PROJECT:
+  description: Project name
+  human_readable: Project
+  required: true
+  data_type: str
+PRSTEP1:
+  description: First processing step
+  human_readable: Processing step
+  required: true
+  data_type: str
+RELEASE:
+  description: Public release date of data
+  human_readable: Release date
+  required: true
+  data_type: date
+RELEASEC:
+  description: Email address of data release administrator
+  human_readable: Release administrator
+  required: true
+  data_type: email
+RESPAPPL:
+  description: Response function applied
+  human_readable: Applid response function
+  required: false
+  data_type: str
+RESOLVPW:
+  description: Resolving power for spectrometric data
+  human_readable: Resolving power
+  required: false
+  data_type: float
+SETTINGS:
+  description: Additional instrument/acquisition settings
+  human_readable: Settings
+  required: false
+  data_type: str
+SLIT_WID:
+  description: "[arcsec] Slit width"
+  human_readable: Slit width
+  required: false
+  data_type: quantity 
+SOLARNET:
+  description: Fully SOLARNET-compliant=1.0, partially=0.5
+  human_readable: Solarnet compatibility
+  required: true
+  data_type: float
+  allowed_values: [1.0, 0.5, 0]
+TELCONFG:
+  description: Telescope configuration
+  human_readable: Telescope configuration
+  required: false
+  data_type: str
+TELESCOP:
+  description: Telescope/Sensor name
+  human_readable: Telescope
+  required: true
+  data_type: str
+TEXPOSURE:
+  description: The integration time for a single exposure
+  human_readable: Single exposure time
+  required: false
+  data_type: float
+TIMESYS:
+  description: Time scale of the time-related keywords.
+  human_readable: Time system
+  required: true
+  data_type: str
+  allowed_values: ["UTC"]
+VERS_CAL:
+  description: Version of calibration pack applied
+  human_readable: Calibration version
+  required: true
+  data_type: str
+VERS_SW:
+  description: Version of software applied
+  human_readable: Software version
+  required: true
+  data_type: str
+VERSION:
+  description: FITS file processing generation/versio
+  human_readable: Processor version
+  required: true
+  data_type: str
+WAVEBAND:
+  description: Human-readable description of the waveband
+  human_readable: Waveband description
+  required: false
+  data_type: str
+WAVECOV:
+  description: The total wavelength coverage if multiple filters (<WAVEMIN1>-<WAVEMAX1>, etc.)
+  human_readable: Wavelength coverage
+  required: false
+  data_type: str
+WAVELTH:
+  description: The characteristic wavelength of the observation
+  human_readable: Characteristic wavelength
+  required: false
+  data_type: float
+WAVEMAX:
+  description: "[Angstrom] Maximum wavelength covered by filter"
+  human_readable: Maximum wavelength
+  required: true
+  data_type: quantity
+WAVEMIN:
+  description: "[Angstrom] Minimum wavelength covered by filter"
+  human_readable: Minimum wavelength
+  required: true
```

Comment:
It seems odd to me that this is required. This is a less-well defined quantity for some types of data, e.g. imaging instruments which have a bandpass or a magnetograph.

Reply:
The recommendations are specific to the kinds of observatories so you are right that this is technically only required for "Spectrographs and filter instruments
(Sections 3.2 and 5.4)" see page 59. I wasn't sure how to implement this kind of level of requirement system.